### ISSUE TITLE
A11y_SQLAzureDataStudio_NewServerGroup_AddServerGroup_Keyboard:Focus is not logical after cancel button.

### ISSUE THREAD

**Issue by fsteffi** [microsoft/azuredatastudio#9224]
[“Check out Accessibility Insights! ](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier.”

#A11y_SQLAzureDataStudio;#A11yMAS;#A11ySev2;#A11yTCS;#SQL Azure Data Studio;#Benchmark;#DesktopApp;#MacOs;#MAS2.4.3;#Keyboard;

**Environment Details:**
Application Name: Azure Data Studio
Version: 1.15.0-insider
Commit: 3a8b74a
Date: 2020-02-11T05:49:41.181Z
VS Code: 1.42.0
Electron: 7.1.11
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Darwin x64 19.3.0
Mac OS:10.15.3

**Additional Details:**
Mac OS:10.15.3
MAS References:MAS2.4.3

**Repro Steps:**
1. Launch Azure Data Studio.
2. Select the Connections icon (if not already selected) from left bar.
3. Hover over the 'Servers' header and select the 'New Server Group' icon.
3. Navigate to Cancel button.

**Actual:**
On navigating to the cancel button hit enter the next tab key focus is seen on the server icon in the left panel.

**Same issue is observed in the following page:**
1.Add new controller

**Expected:**
The focus should be on the button "new server group"  button that opened the dialog.

**User Impact:**
Keyboard users will not receive a logical navigation sequence as well as the button that revealed the content will not be visible on the first tab.

**Attachment link for Reference :**
https://microsoft-my.sharepoint.com/:p:/g/personal/v-stfe_microsoft_com1/EfgAHmo9K-tOjS7Zz1b7JM4BDgyzy8dDOpRN36JS8yy0DA?e=iMBmbh

**Comment by alanrenmsft** [comment]
dupe of #9227

**Comment by BhanuPrakashGit** [comment]
Verified the bug, issue is not repro'ing.
**GithubTags:**#Closed;
PFA for reference
[9224_Cllosed.zip](https://github.com/microsoft/azuredatastudio/files/10742025/9224_Cllosed.zip)

**Comment by mstechie** [comment]
GitHubTags:#SQLADS-MAC-Feb2020;
